op-batcher/batcher/driver.go

```diff
@@ -606,3 +676,60 @@ func logFields(xs ...any) (fs []any) {
 	}
 	return fs
 }
+
+type TestBatchSubmitter struct {
+	bs  *BatchSubmitter
```

Comment:
```suggestion
	*BatchSubmitter
```
If we embed, all methods are automatically available and we don't need to forward e.g. `StartBatchSubmitting`. Or did you deliberately not embed?

Reply:
I see, but what about embedding a pointer?

Reply:
good point, done!